src/Microsoft.ML.PipelineInference/AutoInference.cs

```diff
@@ -98,12 +98,14 @@ private SupportedMetric(string name, bool isMaximizing)
             public static SupportedMetric ByName(string name)
             {
                 var fields =
-                    typeof(SupportedMetric).GetMembers(BindingFlags.Static | BindingFlags.Public)
+                    typeof(SupportedMetric).GetFields(BindingFlags.Static | BindingFlags.Public)
                     .Where(s => s.MemberType == MemberTypes.Field);
```

Comment:
>.Where(s => s.MemberType == MemberTypes.Field); [](start = 20, length = 47)

Given that you're now using `GetFields` is it necessary to have this filter on `s.MemberType == MemberTypes.Field` any longer?

Reply:
Good point, I can likely remove it now.